### PR TITLE
Describe Merknad object and merknadstype code list

### PR DESCRIPTION
**Pull request by plamenbotev**

### CODE REVIEW

**Review by orlinDocumaster — APPROVED** [review]

File: version1/02 - Object types.md [hunk 1]
```diff
@@ -603,3 +610,14 @@ The following notation is used in the tables below:
...
+|   M310    | string    | tekst                  |    x     |             |              |  |  x   |  x   |   x   |   x  |      |        |                                                                                                                                              |
+|   M084    | string    | merknadstype           |    x     |             | merknadstype |  |  x   |  x   |   x   |      |      |        |                                                                                                                                              |
+|           |           |                        |          |             |              |  |      |      |       |      |      |        |                                                                                                                                              |
+|           | ref       | refMappe               |    c     |             |              |  |      |  x   |   x   |      |  x   |        | <p>&bull; references **AbstraktMappe**</p><p>&bull; exactly one of **refMappe**, **Dokument** and **refRegistrering** is required</p>        |
```
Comment (ivaylomitrev) [inline version1/02 - Object types.md]: These comments should say `refDokument` instead of just `Dokument`.

File: version1/03 - Code lists.md [hunk 2]
```diff
@@ -195,3 +195,10 @@ No default values are present in this list and must be configured in the system
 |      |      | &nbsp;  |
 
 No default values are present in this list and must be configured in the system before use.
+
+### merknadstype (M619)
+
+| Code | Name                  | Comment |
+|:-----|:----------------------|:--------|
+| IMER | Intern merknad        |         |
+| MER  | Merknad               |  &nbsp; |
```
Comment (ivaylomitrev) [inline version1/03 - Code lists.md:204]: There should be a new line at the end of the file.

File: version1/02 - Object types.md [hunk 3]
```diff
@@ -400,7 +404,8 @@ The following notation is used in the tables below:
...
 |           | ref       | refSekundaerKlasse            |          |             |                          |  |      |      |   x   |      |  x   |   x    | <p>&bull; references a **Klasse** from any **Klassifikasjonssystem**</p>                                                                                                                    |
 |           | ref       | refEksternId                  |          |             |                          |  |      |      |   x   |      |  x   |        |                                                                                                                                                                                             |
-|           | ref       | refNoekkelord                 |          |             |                          |  |      |      |   x   |      |  x   |        | &nbsp;                                                                                                                                                                                      |
```
Comment (ivaylomitrev) [inline version1/02 - Object types.md:403]: Let's remove the `&nbsp;`s from the rest of the tables too (`AbstraktMappe` and `AbstraktRegistrering`).

**Review by ivaylomitrev — CHANGES_REQUESTED** [review]

File: version1/02 - Object types.md (hunk 1 above)
Comment (plamenbotev) [reply version1/02 - Object types.md]: Changed.

File: version1/02 - Object types.md (hunk 3 above)
Comment (plamenbotev) [reply version1/02 - Object types.md:403]: Removed.

File: version1/03 - Code lists.md (hunk 2 above)
Comment (plamenbotev) [reply version1/03 - Code lists.md:204]: Currently there is one empty line at the end of the file.

**Review by ivaylomitrev — APPROVED** [review]

**Comment by ivaylomitrev** [comment]
@douzounov , would you like to take a look at this before we merge it?

**Pull request merged**